### PR TITLE
fix: ring adjustment in search blocks

### PR DESCRIPTION
**Pull request by aniamusial**
# Related issue

https://vsf.atlassian.net/browse/SFUI2-1204?atlOrigin=eyJpIjoiOTgxYjA1MzQxNDQwNDBmYzk4YmU4YzMwNjMxMjcwNjYiLCJwIjoiaiJ9

# Scope of work

- adjusted rings

# Screenshots of visual changes

# Checklist

- [x] Self code-reviewed
- [x] Changes documented
- [x] Semantic HTML
- [x] SSR-friendly
- [x] Caching friendly
- [x] a11y for WCAG 2.0 AA
- [x] examples created
- [x] blocks created
- [x] cypress tests created

### CODE REVIEW

File: apps/preview/next/pages/showcases/Search/SearchWithButton.tsx [hunk 1]
```diff
@@ -130,7 +130,7 @@ export default function SearchWithIcon() {
           value={searchValue}
           onChange={handleChange}
           onFocus={open}
-          wrapperClassName="w-full ring-0 active:ring-0 hover:ring-0 focus-within:ring-0 border-y border-l border-neutral-200 rounded-r-none hover:border-primary-800 active:border-primary-700 active:border-y-2 active:border-l-2 focus-within:border-y-2 focus-within:border-l-2 focus-within:border-primary-700"
+          wrapperClassName="w-full !ring-0 active:!ring-0 hover:!ring-0 focus-within:!ring-0 border-y border-l border-neutral-200 rounded-r-none hover:border-primary-800 active:border-primary-700 active:border-y-2 active:border-l-2 focus-within:border-y-2 focus-within:border-l-2 focus-within:border-primary-700"
```
Comment (dkacper) [inline apps/preview/next/pages/showcases/Search/SearchWithButton.tsx:133]: You've fully overwritten default behaviour. 🤔 Does it me it's a special case or the default behaviour is wrong?

Comment (aniamusial) [reply apps/preview/next/pages/showcases/Search/SearchWithButton.tsx:133]: its a special case - we cant have "border" on the right due to the button. And ring always applies to all borders, you can't remove "part" of it.

Comment (dkacper) [reply apps/preview/next/pages/showcases/Search/SearchWithButton.tsx:133]: Alright, so it's fine to overwrite the default behaviour. 👍🏼

**Review by dkacper — APPROVED** [review]
LGTM ⭐

**Pull request merged**